neptune/new/handler.py

```diff
@@ -144,5 +258,166 @@ def pop(self, path: str, wait: bool = False) -> None:
         verify_type("path", path, str)
         self._run.pop(join_paths(self._path, path), wait)
 
+    # Following attributes are implemented only for docstring hints and autocomplete
+    DOCSTRING_ATTRIBUTES = [
+        'remove',
+        'clear',
+        'fetch',
+        'fetch_last',
+        'fetch_values',
+        'delete_files',
+        'download',
+        'download_last',
+    ]
+
+    def remove(self, values: Union[str, Iterable[str]], wait: bool = False) -> None:
+        """Fetches all tags from Neptune servers.
+
+        Available for following field types (`Field types docs page`_):
+            * `StringSet`
+
+        Args:
+            values (str or collection of str): Tag or tags to be added.
+            wait (bool, optional): If `True` the client will wait to send all tracked metadata to the server first.
+                This makes the call synchronous.
+                Defaults to `False`.
+
+        .. _Field types docs page:
+           https://docs.neptune.ai/api-reference/field-types
+        """
+        raise NeptuneException('Should be never called.')
+
+    def clear(self, wait: bool = False):
+        """Removes all tags from the `StringSet`.
+
+        Available for following field types (`Field types docs page`_):
+            * `StringSet`
+
+        Args:
+            wait (bool, optional): If `True` the client will wait to send all tracked metadata to the server first.
+                This makes the call synchronous.
+                Defaults to `False`.
+
+        .. _Field types docs page:
+           https://docs.neptune.ai/api-reference/field-types
+        """
+        raise NeptuneException('Should be never called.')
+
+    def fetch(self):
+        """Fetches field value from Neptune servers.
+
+        Available for following field types (`Field types docs page`_):
+            * `Integer`
+            * `Float`
+            * `Boolean`
+            * `String`
+            * `DateTime`
+            * `StringSet`
+
+        Returns:
+            Value stored in the field in form of `set` or sigle value.
```

Comment:
Value stored in the field in form of `set` or sigle value. 
->
Value stored in the field.